Example/RealmResultsController-iOSTests/RealmResultsCacheTests.swift

```diff
@@ -480,7 +485,20 @@ class CacheSpec: QuickSpec {
                     expect(type) == RealmCacheUpdateType.Move
                 }
             }
-           
+
+            context("the object is in the cache and will change sections (where new section doesn't exist yet)") {
+                beforeEach {
+                    initWithKeypath(true)
+
+                    let task = Task()
+                    task.id = resolvedTasks[0].id
+                    task.resolved = false
+                    type = cache.updateType(task)
+                }
+                it("returns type .Move") {
+                    expect(type) == RealmCacheUpdateType.Move
```

Comment:
I think in this case the desired test to have is if a nonResolvedTask `resolved` flag changes to `true`, it should return a `.Move`. In this concrete case it should return a deletion, as the predicate of the `request` is saying that it only "allows" tasks that have `resolved == true`, and this is not the case